src/Web/WebMVC/WebMVC.csproj

```diff
@@ -31,10 +31,9 @@
     <PackageReference Include="Microsoft.AspNetCore.HealthChecks" Version="1.0.0" />
     <PackageReference Include="Microsoft.Extensions.Http.Polly" Version="2.2.0" />
     <PackageReference Include="Microsoft.Extensions.Logging.AzureAppServices" Version="2.2.0" />
-    <PackageReference Include="Microsoft.AspNetCore.App" Version="2.2.0" />
+    <PackageReference Include="Microsoft.AspNetCore.App" Version="2.2.3" />
     <PackageReference Include="Microsoft.AspNetCore.DataProtection.Redis" Version="2.2.0-preview2-35157" />
     <PackageReference Include="Microsoft.VisualStudio.Azure.Fabric.MSBuild" Version="1.6.7" />
-    <PackageReference Include="Microsoft.Web.LibraryManager.Build" Version="1.0.172" />
```

Comment:
Restore LibMan Builder

Reply:
Ok, if it makes sense. However it that case, I think the files array that specifies _the files to be downloaded_ should be removed because this is the second solution until it is fixed

Reply:
I'm sorry, I'm afraid I don't understand.

Why it's necessary to remove the files array?, those are just the files needed and, what has to be fixed regarding this?

Reply:
because during build, the manifest file of libman, shows an error (see #848), which only fixes in this two ways: [see details here](https://github.com/aspnet/LibraryManager/issues/111)

Reply:
@mvelosop, and this can only be resolved by either of the following:
1. Removing LibraryManagerBuild NuGet package (which is not right as you suggested)
2. removing the files array which specifies the files to be downloaded by LibMan
3. manual workflow
   a. putting the files into **.node_nodules** folder, using _filesystem_ as a source, specifying those files from